README.md

```diff
@@ -44,8 +44,20 @@ The config section contains meta-data about the template. All of these attribute
 * name - The unique template name. All lower-case and only the characters a-z, 0-9 and the dash.
 * label - The human-readable template name, used for display.
 * description - An English description of the template, used for display.
+* category - ["blogging" | "hosting" | "infrastructure" | "email" | "e-commerce" | "support" ]
```

Comment:
> It's confusing to me because Heroku could easily be considered "hosting".

Yes, Heroku is an outlier, but I hope we don't have many of these. Infrastructure is for things like CDN, Realtime APIs, link shorteners, etc...

I am OK to move Heroku to hosting if that helps.

> office-365 is also better suited to "productivity"

I like this proposal because google-apps also fits into this category. I am considering merging "marketing" and "support" to the "productivity" category.